settei/presets/celery.py

```diff
@@ -57,9 +57,25 @@ class WorkerConfiguration(LoggingConfiguration):
         """The backend used by Celery to store task results.  See also Celery's
         docs about result backends:
 
-        http://docs.celeryproject.org/en/latest/configuration.html#celery-result-backend
+        https://docs.celeryproject.org/en/master/userguide/configuration.html#std:setting-result_backend
```

Comment:
I think this link is better. (2)
```suggestion
        https://docs.celeryproject.org/en/latest/userguide/configuration.html#std:setting-result_backend
```